pipelines/tekton-bundle-builder/patch.yaml

```diff
@@ -0,0 +1,9 @@
+---
+- op: replace
+  path: /metadata/name
+  value: tekton-bundle-builder
+- op: replace
+  path: /spec/tasks/4/taskRef
+  value:
+    name: tkn-bundle
+    version: "0.1"
```

Comment:
Not sure if it's obvious to others, but how about a comment to explain what the patches are for and why they are needed.